### PR TITLE
Update translations: pt-BR

### PR DESCRIPTION
**Pull request by BRGustavoRibeiro**
Created Brazilian Portuguese (pt-BR) translations for inlang.com.

### CODE REVIEW

**Comment by BRGustavoRibeiro** [comment]
I have read the CLA Document and I hereby sign the CLA

File: project.inlang.json [hunk 1]
```diff
@@ -1,15 +1,26 @@
...
+  "modules": [
+    "https://cdn.jsdelivr.net/npm/@inlang/plugin-json@4/dist/index.js",
+    "https://cdn.jsdelivr.net/npm/@inlang/message-lint-rule-missing-translation/dist/index.js",
+    "https://cdn.jsdelivr.net/npm/@inlang/message-lint-rule-empty-pattern/dist/index.js",
+    "https://cdn.jsdelivr.net/npm/@inlang/message-lint-rule-without-source/dist/index.js",
+    "https://cdn.jsdelivr.net/npm/@inlang/message-lint-rule-identical-pattern/dist/index.js"
+  ],
+  "plugin.inlang.json": {
+    "pathPattern": "./inlang/source-code/website/lang/{languageTag}.json",
+    "ignore": [],
```
Comment (jannesblobel) [inline project.inlang.json:20]: @BRGustavoRibeiro, thanks for your contribution :) 
Could you let me know why you changed the settings? Or did you use the editor, and it changed it?

Comment (BRGustavoRibeiro) [reply project.inlang.json:20]: I only used the web editor - it probably changed it automatically.

Comment (jannesblobel) [reply project.inlang.json:20]: Oh, sorry, I also encountered this bug myself and forgot to tell you :D It's a 'bug' on the Editor side. 

by the way, I have read the thread about your app. If you need a German translation, please feel free to ping me any time. :)

**Comment by jannesblobel** [comment]
Hey @BRGustavoRibeiro, 
Perhaps you noticed that we did a redesign of our website and also dogfooded our products 🥳 
Unfortunately,  your fantastic contribution is gone, and our Website is not available in Pt-br anymore :( 
I know it is not self-evident, but I would ask you to translate inlang again into Brazilian Portuguese. ❤️ 
The current status is not justifiable.

https://inlang.com/editor/github.com/inlang/monorepo?lang=en&lang=de&lang=pt-BR&lang=sk&lang=zh

**Pull request merged**